File: src/2309.py
s = list(map(int, [input() for i in range(9)]))
tmp = [i for i in s]
res = []
print(s)
for i in range(8):
    for j in range(i + 1, 9):
        print(s[i])
        tmp.remove(s[i])
        tmp.remove(s[j])
        if sum(tmp) == 100:
            res = tmp
            break
        tmp = [i for i in s]

print(res)

File: src/2309.js
n=require('fs').readFileSync('dev/stdin').toString().trim().split('\n').map(v=>+v).sort((a,b)=>a-b);
(()=>{
  for (let i=0; i<8; i++) {
    for (let j=i+1;j<9;j++) {
      let tmp=[...n]
      tmp.splice(j,1)
      tmp.splice(i,1)
      let sum=tmp.reduce((t,v)=>t+v,0)
      if (sum===100) {
        console.log(tmp.join`\n`)
        return
      }
    }
  }
})()
